setup.py

```diff
@@ -7,7 +7,7 @@
       author='Red Hat Application SRE Team',
       author_email="sd-app-sre@redhat.com",
       description='',
-      python_requires='>=3.7',
+      python_requires='>=3.8',
```

Comment:
3.9 here as well ?

Reply:
You are right. Actually it is better to keep that in sync with base images :+1:

Reply:
Thanks.. i hope better python experts will not contradict now ;-)